pkg/controller/machinedeployment/machinedeployment_controller_test.go

```diff
@@ -67,36 +76,33 @@ func TestReconcile(t *testing.T) {
 	g.Expect(err).NotTo(gomega.HaveOccurred())
 	defer c.Delete(context.TODO(), instance)
 	g.Eventually(requests, timeout).Should(gomega.Receive(gomega.Equal(expectedRequest)))
+	g.Eventually(errors, timeout).Should(gomega.Receive(gomega.BeNil()))
 
-	/*
-		// Verify that the MachineSet was created.
-		machineSets := &clusterv1alpha1.MachineSetList{}
-		g.Eventually(func() int {
-			if err := c.List(context.TODO(), &client.ListOptions{}, machineSets); err != nil {
-				return -1
-			}
-			return len(machineSets.Items)
-		}, timeout).Should(gomega.BeEquivalentTo(1))
-		ms := machineSets.Items[0]
-		g.Expect(ms.Spec.Replicas).Should(gomega.BeEquivalentTo(1))
-		g.Expect(ms.Spec.Template.Spec.Versions.Kubelet).Should(gomega.Equal("1.10.3"))
-
-		// Verify that Machines were created with the desired kubelet version.
-		// TODO: remove this?
```

Comment:
I've removed this because`mgr` only has a `Reconciler` for `machineDeployments`, hence there is nothing that could create a `machine`.

Also IMHO the tests here should not test the `machineSet` controller

Reply:
sgtm